feat: allow the swagger path to be configurable

Junjiequan opened the pull request:
## Description

This PR makes the Swagger path configurable by allowing it to be set through the SWAGGER_PATH environment variable. If not provided, it defaults to “explorer.”

## Motivation

Making the Swagger path configurable improves flexibility, allowing different paths for different environments 

## Fixes:
Please provide a list of the fixes implemented by this PR

* Items added

## Changes:
Please provide a list of the changes implemented by this PR

* changes made

## Tests included

- [ ] Included for each change/fix?
- [ ] Passing? (Merge will not be approved unless this is checked) 

## Documentation
- [ ] swagger documentation updated \[required\]
- [ ] official documentation updated \[nice-to-have]

### official documentation info
If you have updated the official documentation, please provide PR # and URL of the pages where the updates are included

Junjiequan committed:
feat: allow the swagger path to be configurable

nitrosx commented on src/main.ts:
```diff
@@ -36,8 +36,9 @@ async function bootstrap() {
       docExpansion: "none",
     },
   };
+  const swaggerPath = process.env.SWAGGER_PATH || "explorer";
```
Can you place this in the config service, please?

Junjiequan replied:
fixed

nitrosx reviewed (changes requested):
Good to go, just one minor request

Junjiequan committed:
Update the version in package.json from 4.0.0 to 4.5.0.
In configuration.ts, add a new property "versions.api" with a default value of "v3".
Also, add a new property "swaggerPath" with a default value of "explorer".

Junjiequan committed:
make api version not configurable.

nitrosx commented on src/config/configuration.ts:
```diff
@@ -58,6 +58,10 @@ const configuration = () => {
   const config = {
+    VERSIONS: {
+      API: "v3",
+    },
```
Do you mind using camel case, so we are consistent with all the other entries?

Junjiequan replied:
fixed

nitrosx commented on src/main.ts:
```diff
@@ -15,6 +15,11 @@ async function bootstrap() {
+  const configService: ConfigService<Record<string, unknown>, false> = app.get(
+    ConfigService,
+  );
+  const apiVersion = configService.get<string>("VERSIONS.API");
```
Camel case?

Junjiequan replied:
fixed

nitrosx reviewed (changes requested):
To minor changes

Junjiequan committed:
converting api version vars naming to lower case
